build-support/bin/check_rust_pre_commit.sh

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env bash
 
 # NB: pre-commit runs in the context of GIT_WORK_TREE, ie: pwd == REPO_ROOT
+source build-support/common.sh
```

Comment:
style nit: other paths in the file use `./build-support` instead of `build-support`